packages/core/src/text/Display.tsx

```diff
@@ -12,7 +12,7 @@ export const Display1 = forwardRef<
   return (
     <Text
       as="span"
-      className={clsx(className, withBaseName(`display1`))}
+      className={clsx(className, withBaseName("display1"))}
```

Comment:
I know this was here already but isn't this equivalent to styleAs="display1" ?

Reply:
yes they are, good spot i'll update these to use styleAs to encourage APi use